src/main/java/mods/eln/server/ServerEventListener.java

```diff
@@ -172,4 +172,16 @@ public static void writeToEaWorldNBT(NBTTagCompound nbt, int dim) {
 
     }
 
+    private class FileNames {
+        final Path worldSave;
+        final Path tempSave;
+        final Path backupSave;
+
+        FileNames(WorldEvent e) {
+            String saveName = getEaWorldSaveName(e.world);
```

Comment:
The method getEAWorldSaveName() could be part of this class, it is only used here.